src/simulation/dynamics/DynOutput/boreAngCalc/_UnitTest/test_bore_ang_calc_inertial_heading.py

```diff
@@ -84,6 +84,29 @@ def bore_ang_calc_inertial_heading_func(show_plots, inertialHeading, eulerLoc):
     TotalSim.InitializeSimulation()
```

Comment:
:small_red_triangle_down: Squash into the previous commit.

Reply:
Dones